src/parser/jobs/rdm/modules/Gauge.tsx

```diff
@@ -0,0 +1,307 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import JOBS from 'data/JOBS'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import {Statistics} from 'parser/core/modules/Statistics'
+import Suggestions, {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {DualStatistic} from 'parser/jobs/rdm/statistics/DualStatistic'
+import React, {Fragment} from 'react'
+import {isSuccessfulHit} from 'utilities'
+
+interface GaugeModifier {
+	white: number
+	black: number
+}
+
+export class Gauge extends CoreGauge {
+	static override title = t('rdm.gauge.title')`Mana Gauge Usage`
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+	@dependency private statistics!: Statistics
+
+	private _whiteManaGauge = this.add(new CounterGauge({
```

Comment:
TypeScript style rule: Do not use _ for private properties; we use that convention for JavaScript because there's no accessibility modifier, but it's not necessary or desired in TypeScript files.

Reply:
updated with suggestion.